<?xml version="1.0"?>

<launch>
    <arg name="darknet_topic" default="darknet_ros/boundingbox"/>
    <arg name="retinanet_topic" default="retina_ros/boundingbox"/>
    <arg name="pcl_topic" default="points2"/>
    <arg name="fov_topic" default="fov_regions"/>
    <arg name="target_frame" default="odom"/>
    <arg name="target_class" default="chair"/>
    <arg name="visual" default="true"/>
    <arg name="pcl_visual" default="true"/>

    <node pkg="pointcloud_processing" type="sep_processing" name="sep_processing_node" output="screen">
        <param name="DARKNET_TOPIC" value="$(arg darknet_topic)"/>
        <param name="PCL_TOPIC" value="$(arg pcl_topic)"/>
        <param name="TARGET_FRAME" value="$(arg target_frame)"/>
        <param name="TARGET_CLASS" value="$(arg target_class)"/>
        <param name="VISUAL" value="$(arg visual)"/>
        <param name="PCL_VISUAL" value="$(arg pcl_visual)"/>
    </node>
</launch>
